demos/common/cpp_gapi/utils_gapi/include/utils_gapi/ov_backend.hpp

```diff
@@ -32,10 +32,25 @@ struct BackendApplicator<ExecNetwork,
                 config.deviceName                    // device specifier
             }.cfgNumRequests(config.maxAsyncRequests)
             .cfgPluginConfig(config.getLegacyConfig());
-        // E#96606
-        if (!config.mean_values.empty() || !config.scale_values.empty()) {
-            throw std::runtime_error(backend.name + " backend doesn't support neither `mean_values` nor `scale_values` at the moment");
+            
+        if (!config.mean_values.empty() && !config.scale_values.empty()) {
+            std::vector<float> means;
+            split(config.mean_values, ' ', means);
+
+            std::vector<float> scales;
+            split(config.scale_values, ' ', scales);
```

Comment:
These two lines can be part of `main.cpp` not to repeat them for every backend params